chapters.py

```diff
@@ -9,222 +9,255 @@
-        "Type walk to walk around & swim to swim across. " +
+        "\nYou have come across a river. You can either walk around it or swim across it.\n"
+        "Type \"walk/w\" to walk around or \"swim/s\" to swim across. " +
         Fore.LIGHTMAGENTA_EX).lower()
     # if user inputs swim then
-    if answer == "swim":
+    if answer == "swim" or answer == "s":
         game_over(
             Fore.RED +
-            "You swam across the river and were eaten by an aligator \U0001F480"
+            "You swam across the river and were eaten by an alligator \U0001F480"
         )
     # if user inputs walk then
-    elif answer == "walk":
+    elif answer == "walk" or answer == "w":
         # q2
         answer = input(
             Fore.GREEN +
-            "You walked for many miles, ran out of water and remembered "
-            "that there was a shop far away (10 miles/16kms) which supplies water."
-            "Do you want to go there (yes/no)? " +
+            "You have walked many miles and run out of water. "
+            "Fortunately, there is a shop located 10 miles (16kms) away that supplies water. "
+            "Would you like to go there? (yes/no): " +
             Fore.LIGHTMAGENTA_EX).lower()
         # if user inputs no then
-        if answer == "no":
+        if answer == "no" or answer == "n":
             game_over(
                 Fore.RED +
-                "You were very de-hydrated and died of thirst when you were walking. \U0001F480"
+                "You died of dehydration while walking due to severe lack of water. \U0001F480"
             )
 
         # if user inputs yes then
-        elif answer == "yes":
+        elif answer == "yes" or answer == "y":
             print(
                 Fore.GREEN +
-                "You went 10 miles walking and bought 10 liters of drinking water. "
+                "You walked 10 miles and bought 10 liters of drinking water. "
             )
             # q3
             answer = input(
                 Fore.GREEN +
-                "You are thirsty, do you want to drink some water (yes/no)? " +
+                "You are thirsty. Would you like to drink some water? (yes/no) : " +
                 Fore.LIGHTMAGENTA_EX).lower()
         # if user inputs yes then
-        if answer == "yes":
+        if answer == "yes" or answer == "y":
             print(Fore.GREEN +
-                  "You drank 5 liters of water and now you feel refreshed.")
+                  "After drinking 5 liters of water, you feel refreshed.")
 
         # if user inputs no then
-        elif answer == "no":
-            game_over(Fore.RED + "You died of thirst.\U0001F480 ")
+        elif answer == "no" or answer == "n":
+            game_over(Fore.RED + "You have died of thirst.\U0001F480 ")
         else:
-            print(Fore.RED + "Not a valid answer. You die. \U0001F480")
+            print(Fore.RED + "I'm sorry, I don't understand the input. You Dead. \U0001F480")
```

Comment:
```suggestion
            print(Fore.RED + "I'm sorry, I don't understand the input. You Died. \U0001F480")
```

You dead isn't proper grammar, it should be you died. Please make the edits in all relevant places.